src/shared/agent_op.c

```diff
@@ -619,8 +621,17 @@ static cJSON* w_create_agent_add_payload(const char *name,
         cJSON_AddStringToObject(arguments, "id", id);
     }
 
-    if (force >= 0) {
-        cJSON_AddNumberToObject(arguments, "force", force);
+    if (force_options) {
+        cJSON* j_force = cJSON_CreateObject();
+        cJSON* j_disconnected_time = cJSON_CreateObject();
+        cJSON_AddItemToObjectCS(j_force, "disconnected_time", j_disconnected_time);
```

Comment:
Is it really desired to use **cJSON_AddItemToObjectCS** instead of **cJSON_AddItemToObject**?

Reply:
Only for reading purposes, I suggest to order **cJSON_AddItemToObjectCS(j_force, "disconnected_time", j_disconnected_time)**; together with **cJSON_AddBoolToObject(j_disconnected_time, "enabled", force_options->disconnected_time_enabled);** and  **cJSON_AddNumberToObject(j_disconnected_time, "value", force_options->disconnected_time);**

Reply:
Fixed!